sound/soc/sof/core.c

```diff
@@ -288,35 +298,51 @@ static int sof_probe(struct platform_device *pdev)
 		goto dbg_err;
 	}
 
+	/*
+	 * let's register miscdev now.
+	 * This will be used by the device manager in hypervisor
+	 * so that vFE can communicate with vBE
+	 */
+	sof_virtio_miscdev_register(sdev);
```

Comment:
I think we only need to do this if vBE support has been selected by kconfig ?

Reply:
Yes. And I define static inline function to do nothing in the header file if vBE is not selected.

Reply:
ok, change comment to "optionally register virtio miscdev "

Reply:
OK.

Reply:
got it